### ISSUE TITLE
add right click of windows

### ISSUE THREAD

**Issue by Chetan-KK** [Chetan-KK/Windows-10-Clone#8]
replicate right click of windows in this website

**Comment by FurrukhJamal** [comment]
do you have screens of what it looked like in the real windows?

**Comment by Chetan-KK** [comment]
here it is:

![Screenshot (165)](https://github.com/Chetan-KK/Windows-10-Clone/assets/44507196/7f7188c8-dc3a-43a9-b67c-1d1aedb70f93)

don't create git and graphics options

![sort](https://github.com/Chetan-KK/Windows-10-Clone/assets/44507196/a29d638a-9ee0-470b-adb7-77d0d2c9cb73)
![new](https://github.com/Chetan-KK/Windows-10-Clone/assets/44507196/113fc65f-987f-42ec-b797-0935a10a2556)
![view](https://github.com/Chetan-KK/Windows-10-Clone/assets/44507196/d9828fb1-b181-47f6-82bf-e0b2c7111140)

**Comment by Youssef11296** [comment]
@Chetan-KK, I'm open to contributing with you, bruh.
If this issue is still opened and not assigned, feel free to assign it to me.

**Comment by FurrukhJamal** [comment]
yeah I got busy with my personal project, will get back to this once I am done, if @Youssef11296 wants to do it you can have him do this, I will look into other issues when I get the time

**Comment by Chetan-KK** [comment]
@Youssef11296 feel free to contribute and make pull request, i can't assign it to you but i will merge when it's done

**Comment by FurrukhJamal** [comment]
is this been done?

**Comment by Chetan-KK** [comment]
no, it's not

**Comment by FurrukhJamal** [comment]
now that I have just spend a couple minutes thinking abt it, it is not as simple a task as it seems like, there are at least 6 corner cases for a right click when the mouse is at 6 different points on the screen

**Comment by FurrukhJamal** [comment]
hey what is the work flow like, so that I have the latest version of the original repo locally, do i fork again or just git fetch on my main branch?

**Comment by Chetan-KK** [comment]
click sync button on your forked repo to make it up to date, then use git pull to get all changes locally

**Comment by FurrukhJamal** [comment]
is the latest updated version working correctly? I just updated my fork pulled and it broke at my side giving 
 
`[vite] Internal server error: [BABEL] D:\projects\opensourceprojects\Windows-10-Clone\src\main.jsx: Cannot find module 'D:\projects\opensourceprojects\Windows-10-Clone\node_modules\@babel\helper-function-name\lib\index.js'. Please verify that the package.json has a valid "main" entry`

**Comment by FurrukhJamal** [comment]
never mind, I messed up my local branch, cloning again and doing it in another directory
